charts/iot-agent/README.MD

```diff
@@ -53,6 +53,8 @@ chart and their default values.
 | `priorityClassName`               | If specified, indicates the pod's priority  | ``                                                  |
 | `affinity`               | If specified, defines affinity rules  | ``                                                  |
 
+Be careful with enabling `multiCore` on k8s running on top of aws nodes.
```

Comment:
perhaps we should explain why or at least what to watch out for?

Reply:
processors a constantly spawn and killed, and the service is not working...

Reply:
holy moly!